tests/cli/commands/test_celery_command.py

```diff
@@ -69,6 +70,7 @@ class TestCeleryStopCommand:
     @classmethod
     def setup_class(cls):
         with conf_vars({("core", "executor"): "CeleryExecutor"}):
+            importlib.reload(executor_loader)
```

Comment:
We do more caching in the executor loader now, use `reload` to get fresh module state. Similar to cli_parser.